ui/components/multichain/notification-list-item/notification-list-item.stories.tsx

```diff
@@ -135,5 +135,5 @@ NftSentNotification.args = {
       },
     ],
   },
-  amount: '#1234',
+  amount: '1234',
```

Comment:
Removing hash here to prevent false positive

Reply:
perhaps stories should be excluded from the rule

Reply:
Good point. While I did consider this, I believe that stories shouldn't be exempt from best practices; rather, they should serve as a reflection of them.

Reply:
Updated here https://github.com/MetaMask/metamask-extension/pull/24089/commits/25bf40ac95343b1634bcfa742a55d77f535f6f80